### PR TITLE
Add recipe for french toast

### PR DESCRIPTION
**Pull request by githubstudent**
I think we should have a recipe for french toast. What do you think?

### CODE REVIEW

**Comment by githubstudent** [comment]
@githubteacher can you review this new recipe?

File: _posts/french-toast.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# French Toast
+
+1. 4 eggs
```
Comment (githubteacher) [inline _posts/french-toast.md]: Can we make this 3 eggs instead?

File: _posts/french-toast.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+2. 1/2 Cup milk
+3. Some bread (sliced)
```
Comment (githubteacher) [inline _posts/french-toast.md]: “Some bread” is pretty vague, maybe we should say how many slices?

**Comment by githubstudent** [comment]
@githubteacher I have addressed your feedback, what do you think?

@matthewmccullough any thoughts from you here?

**Comment by matthewmccullough** [comment]
Yes. I like these changes.

**Pull request merged**